extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputFormat.java

```diff
@@ -49,8 +45,6 @@ public class OpenCensusProtobufInputFormat implements InputFormat
   private final String metricLabelPrefix;
```

Comment:
VERSION_HEADER_KEY and OPENTELEMETRY_FORMAT_VERSION constants above are no longer used and can be removed

Reply:
I thought I had removed those. Good catch. Ran the inspection too